REQUIRE

```diff
@@ -4,7 +4,7 @@ Distributions
 Docile
 DualNumbers
 FITSIO
-ForwardDiff 0.0.3 0.1
+ForwardDiff # Currently only required for testing.
```

Comment:
I wonder if we could use ForwardDiff.jl for testing the derivatives too, rather than relying on GSL.jl. That library throws a lot of errors/warnings with Julia 0.4 that I don't think will get fixed any time soon, and I worry finite differences could miss discrepancies...it's tricky to set the tolerance high enough to avoid cating rounding errors, but low enough not to miss real errors.